### ISSUE TITLE
[studio-ui] The UI is sending null as site name

### ISSUE THREAD

**Issue by alhambrav** [craftercms/craftercms#4003]
## Describe the bug
 There's an error in the logs after logging in from a fresh install.  The UI is sending null as site name.

```
[ERROR] 2020-04-28T12:00:14,165 [http-nio-8080-exec-5] [blob.BlobAwareContentRepository] | Error getting content /config/studio/site-config.xml in site null
org.craftercms.studio.api.v1.exception.ContentNotFoundException: Repository not found for site null
```
## To Reproduce
1. git clone craftercms then build, deploy and start
2. Login to Studio
3. Notice the error message in the tomcat logs

## Expected behavior
There should be no errors in the tomcat log when logging in

## Screenshots
{{If applicable, add screenshots to help explain your problem.}}

## Logs
https://gist.github.com/alhambrav/3c0ee478e3747a8478467e7127f69376

### Version
Crafter CMS v3.2.0-SNAPSHOT

### OS
macOS

### Browser
Chrome browser

## Additional context
{{Add any other context about the problem here.}}

**Comment by rart** [comment]
Fixed on d7ea872846831d0fc4709165a5547b348685e95a

**Comment by alhambrav** [comment]
Verified fixed
